pkg/platform/model/buildplanner.go

```diff
@@ -361,16 +364,45 @@ func processBuildPlannerError(bpErr error, fallbackMessage string) error {
 	return errs.Wrap(bpErr, fallbackMessage)
 }
 
+var versionRe = regexp.MustCompile(`^\d+(\.\d+)+$`)
+
+func isExactVersion(version string) bool {
+	return versionRe.MatchString(version)
+}
+
 func isWildcardVersion(version string) bool {
 	return strings.Index(version, ".x") >= 0 || strings.Index(version, ".X") >= 0
 }
 
 func VersionStringToRequirements(version string) ([]bpModel.VersionRequirement, error) {
-	if !isWildcardVersion(version) {
+	if isExactVersion(version) {
 		return []bpModel.VersionRequirement{{
 			bpModel.VersionRequirementComparatorKey: "eq",
 			bpModel.VersionRequirementVersionKey:    version,
-		}}, nil // exact version
+		}}, nil
+	}
+
+	if !isWildcardVersion(version) {
+		// Ask the Platform to translate a string like ">=1.2,<1.3" into a list of requirements.
+		// Note that:
+		// - The given requirement name does not matter; it is not looked up.
+		// - The given language only needs to exist. It has no bearing on the translation and is not
+		//   present in the requirements read and returned.
+		dummyLanguage := language.Python3.Requirement()
```

Comment:
Can we actually provide this the name "dummy" or it has to be Python?

Can you make sure with Dashboard-squad that this logic is safe?

Reply:
We need to provide a valid language name. I tried "dummy" and it didn't work.

I confirmed with DS that this is safe. Their response was: "That service is unaware of any real world packages, so your example should be ok.  I can't promise that it won't change in the future to be smarter/interact with our ingredients DB to validate pcakage names, but I doubt that will happen, and if it did, it would likely fall under a "bad request" type response which you are going to handle any way."